src/integTest/resources/liberty-test/install_liberty_javaee7.gradle

```diff
@@ -0,0 +1,38 @@
+buildscript {
+    repositories {
+        mavenLocal()
+        mavenCentral()
+        maven {
+            name = 'Sonatype Nexus Snapshots'
+            url = 'https://oss.sonatype.org/content/repositories/snapshots/'
+        }
+    }
+    dependencies {
+        classpath 'net.wasdev.wlp.gradle.plugins:liberty-gradle-plugin:1.1-SNAPSHOT'
+    }
+}
+
+def test_mode = System.getProperty('runit')
+def wlpLicense = System.getProperty('wlpLicense')
+def wlpVersion = System.getProperty('wlpVersion')
+def WLP_DIR = System.getProperty('wlpInstallDir')
+
+apply plugin: 'liberty'
+
+repositories {
+    mavenCentral()
```

Comment:
FYI, the 'buildscript -> repositories' is to use external library according to doc - 'If your build script needs to use external libraries, you can add them to the script's classpath in the build script itself.'

The latter 'reopsitories' is for adding dependencies - this is what we're using for this issue.